cdf_dump/README.md

```diff
@@ -0,0 +1,186 @@
+# cdf_dump CLI tool
+
+The cdf_dump utility is a simple CLI wrapper for [Cognite Spark Data Source](https://github.com/cognitedata/cdp-spark-datasource)
+to simplify copying of large amounts data from CDF.
+
+## Installation
+
+Download the cdf_dump.jar from https://github.com/cognitedata/cdp-spark-datasource/releases and run it from command line: `java -jar cdf_dump.jar`. The cdf_dump.jar file contains all dependencies, including Apache Spark, so it's bit large, but should not require anything except the JVM on your OS.
+Spark currently only supports Java 11 and Java 8, so cdf_dump also won't work on other JVM versions.
+If you have the choice, we recommend Java 11. You may use GraalVM, but it doesn't seem to improve performance much.
+
+
+## Examples
+
+```shell
+cdf_dump --raw my-db.table -o out_directory
+```
+
+Copies my-db.table into `out_directory/raw/my-db/table` directory in JSON format. You can use [`jq`](https://stedolan.github.io/jq/) to process the data further.
```

Comment:
idk why suggestions don't work here, maybe this could also have been `my-db.table` instead of my-db.table?